apps/meteor/app/lib/server/functions/insertMessage.ts

```diff
@@ -32,8 +32,6 @@ export const insertMessage = async function (
 		}
 		await Messages.insertOne({
```

Comment:
this insertOne is supposed to only run when `existingMessage` is undefined - can you add an `else` block around it?
as it is right now it'll be duplicating messages if the import is executed multiple times.

Reply:
Thankss, fixed!